tiledb/sm/subarray/subarray.h

```diff
@@ -390,6 +532,9 @@ class Subarray {
   /** Returns `true` if at least one dimension has non-default ranges set. */
   bool is_set() const;
 
+  /** Returns number of non-default (set) ranges */
+  int32_t count_set() const;
```

Comment:
Put the word `range` somewhere in this function name. I know this compliments the existing `is_set` but I think that function name is too short too.

Reply:
now 'count_set_ranges()'